src/benchmarks/micro/Properties/launchSettings.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Is this file part of the change or just a VS one?

Reply:
This is the file used to store launch settings (like command line arguments & environment variables) for testing.  Other people will probably use it.

Reply:
But every person will have their own launch settings and will want to set them locally. As a rule we do not check-in things like this.

Reply:
We should add this to the .gitignore file as I think VS autogenerates.

Reply:
Added the change to .gitignore. You can check to see if I got the syntax right